### DISCUSSION TITLE
Problem with hitbox

### DISCUSSION THREAD

**Discussion by EmporiumDevelopment** [diamond-gold/MiniBosses#38] [General]
Yes i have configured the width height scale etc if the scale is set to 1 the entity is normal size but hit box is only half of the entities height, if i set the scale to 2 the full body of the entity is hittable, even if i keep the scale at 1 and set the height to 200 the hit box is still only half of the entities body

**Comment by diamond-gold** [comment]
Are you using the config?

**Comment by EmporiumDevelopment** [comment]
yes

**Comment by diamond-gold** [comment]
Bug reproduced #39
